src/interract.js

```diff
@@ -0,0 +1,19 @@
+// Checkbox function
+export function checkvalue(list) {
+  document.addEventListener('change', (e) => {
+    if (e.target.classList.contains('to-do')) {
+      const index = e.target.dataset.listIndex - 1;
+      e.target.parentNode.children[1].classList.toggle('checked');
+
+      list.checked(index);
+    }
+  });
```

Comment:
- [ ] Great work implementing the changes requested by the previous reviewer. Your checked box is now working fine as expected but there is still a little more you can do to make it better and also give your users the best experience ever. 
- [ ] Currently, when a user toggles the `checkbox` to update the `status` of their todo task it works but the updated `status` is not being stored in the `localStorage`  and due to that when a user toggles a task as `completed: true` and refreshes the page the todo status goes back to its initial state (`completed: false`)

Reply:
Done.